_includes/activity/github-desktop/01-create-repository.md

```diff
@@ -1,8 +1,18 @@
 Now that you have what you need for Git and GitHub installed locally, let's create the repository that will hold your new website.
 
 1. On GitHub.com, [create a new repository](https://github.com/new).
-1. Name your repository `YOUR-USERNAME.github.io`. Why the funny name? Click the **Tell me why** section below to learn more.
+1. Name your repository.
```

Comment:
@beardofedu it feels weird to tell them to do this when they can literally call it anything they want. Should we add some context here such as "this will appear in the URL for your website"?